File: src/main/java/no/nav/pensjon/regler/domain/kode/GPSatsTypeCti.java
package no.nav.pensjon.regler.domain.kode;

public class GPSatsTypeCti extends TypeCti {

    private static final long serialVersionUID = -143021596183424410L;

    public GPSatsTypeCti() {
        super();
    }

    public GPSatsTypeCti(String kode) {
        super(kode);
    }

    public GPSatsTypeCti(GPSatsTypeCti grunnlagsrolleCti) {
        super(grunnlagsrolleCti);
    }
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/kode/GPSatsTypeCti.kt
package no.nav.pensjon.regler.domain.kode

class GPSatsTypeCti : TypeCti {
    constructor() : super()
    constructor(kode: String?) : super(kode)
    constructor(grunnlagsrolleCti: GPSatsTypeCti) : super(grunnlagsrolleCti)
}